bricks.html

```diff
@@ -0,0 +1,21 @@
+<html>
+<body>
+<script src="bricks.js"></script>
+<!-- <div style="width: 100px; height: 50px">
```

Comment:
delete these commented out lines

Reply:
as a general principle: git is the thing we use to save history. So if you want to save something, commit it, then delete it.